package/yast2-services-manager.changes

```diff
@@ -1,8 +1,17 @@
 -------------------------------------------------------------------
+Mon Nov 23 09:50:44 CET 2015 - locilka@suse.com
+
+- Service state 'activating' is considered to be 'active',
+  'deactivating' is 'deactive' and 'reloading' is 'active'
```

Comment:
s/deactive/inactive/
Also, this describes the internal workings, but I would like to see the user's perspective too:
"This fixes the second stage of AutoYaST installation getting hung in certain cases as it tried to restart itself" ?